docs/pages/style-spec.js

```diff
@@ -1392,6 +1301,7 @@ export default class extends React.Component {
                                                 </span>
                                                 {type.map((overload, i) =>
                                                     <div key={i}>{highlightJavascript(renderSignature(name, overload))}</div>)}
+                                                {sdkSupport && <div className='space-bottom2'><SDKSupportTable {...sdkSupport} /></div>}
```

Comment:
These tables are being rendered flush against the preceding code block -- maybe add `space-top2` here also?